### ISSUE TITLE
Suggestion: add plugin before other plugins

### ISSUE THREAD

**Issue by jellymann** [artberri/rollup-plugin-istanbul#6]
I've come across errors resembling the following:
```
08 05 2016 22:47:59.218:ERROR [coverage]: TypeError: Cannot read property 'text' of undefined
    at PROJECT_ROOT/node_modules/istanbul/lib/report/html.js:241:45
```

I have found that placing the istanbul plugin _before_ the other plugins fixes the issue.

Causes Error:
```js
// karma.conf.js
//...
    rollupPreprocessor: {
      rollup: {
        plugins: [
          // ... other plugins
          require("rollup-plugin-babel")(),
          require("rollup-plugin-istanbul")({
            exclude: ["test/**/*.js"]
          })
        ]
      }
    },
//...
```

Fixes Error:
```js
// karma.conf.js
//...
    rollupPreprocessor: {
      rollup: {
        plugins: [
          require("rollup-plugin-istanbul")({
            exclude: ["test/**/*.js"]
          }),
          // ... other plugins
          require("rollup-plugin-babel")()
        ]
      }
    },
//...
```

@artberri Perhaps, if you think this is a good idea, the README ought to updated to reflect this?

**Comment by artberri** [comment]
I've merged your PR #5 , so, the readme now reflects it. Thank you and sorry for being so late.

**Comment by jlmakes** [comment]
This caused unexpected token errors for me, specifically having `rollup-plugin-istanbul` before my ES2015 transpiler; I don't have other plugins, so I can speak to that, but certainly transpiling had to occur first.
